moped-editor/src/views/projects/projectView/ProjectTeam/TeamAutocompleteComponent.js

```diff
@@ -49,7 +50,10 @@ const TeamAutocompleteComponent = ({
     apiRef.current.setEditCellValue({
       id,
       field: "moped_workgroup",
-      value: { workgroup_id: newValue?.workgroup_id },
+      value: {
+        workgroup_id: newValue?.workgroup_id,
+        workgroup_name: workgroupLookup[newValue?.workgroup_id],
```

Comment:
this fixed a bug thats currently in prod where if you edit a row and click a new user and then in the same edit session click the old user again and save that, you would see [object Object] in the workgroup name field. also in other cases where you are actually saving a new user, [object Object] would just flash on the screen in the time between clicking save and [this hook](https://github.com/cityofaustin/atd-moped/blob/37752cc60f54d49fb19ec077d23c2165b5e28915/moped-editor/src/views/projects/projectView/ProjectTeam/ProjectTeamTable.js#L219) running which manually adds the `workgroup_name` to the row data

Reply:
Nice! Thank you for catching AND fixing this bug. 🚀 I was able to see the native object peek out when I adjusted my connection speed. 🙏

FYI @chiaberry this probably affects your work on reusable autocompletes that have dependent fields

Reply:
Thanks for addressing this bug! I am going to check the milestones table to see if I see that same bug there.